src/asammdf/gui/dialogs/advanced_search.py

```diff
@@ -174,7 +174,7 @@ def search_text_changed(self):
                                     (group_index, channel_index): {
                                         "names": [ch.name],
                                         "comment": extract_xml_comment(ch.comment).strip(),
-                                        "unit": ch.conversion and ch.conversion.unit or ch.unit,
+                                        "unit": ch.conversion.unit if ch.conversion else ch.unit,
```

Comment:
same as above, also applies to the following commit diffs

Reply:
Fixed